src/main/java/uk/gov/hmcts/probate/services/submit/services/SubmitService.java

```diff
@@ -60,34 +60,24 @@ public SubmitService(MailClient mailClient, PersistenceClient persistenceClient,
     public JsonNode submit(SubmitData submitData, String userId, String authorization) {
         Optional<CcdCaseResponse> caseResponseOptional = getCCDCase(submitData, userId, authorization);
         FormData formData = persistenceClient.loadFormDataById(submitData.getApplicantEmailAddress());
+
         if (!caseResponseOptional.isPresent()) {
-            if (formData.getSubmissionReference() != 0) {
-                return new TextNode(DUPLICATE_SUBMISSION);
```

Comment:
You could still return the DUPLICATE_SUBMISSION if caseResponseOptional.isPresent() is true?